apps/beeswax/src/beeswax/server/hive_server2_lib_tests.py

```diff
@@ -298,7 +298,7 @@ def test_get_table_with_error(self):
         client.get_table(database='database', table_name='table_name')
       except QueryServerException as e:
         if sys.version_info[0] > 2:
-          req_string = "TGetTablesReq(sessionHandle=TSessionHandle(sessionId=THandleIdentifier(guid=b'1', secret=b'1')), catalogName=None, schemaName='database', tableName='table_name', tableTypes=None)"
+          req_string = "TGetTablesReq(sessionHandle=TSessionHandle(sessionId=THandleIdentifier(guid=b'l\\xc4', secret=b'l\\xc4')), catalogName=None, schemaName='database', tableName='table_name', tableTypes=None)"
```

Comment:
and for this, 
<img width="1789" alt="Screenshot 2021-01-12 at 11 43 33 PM" src="https://user-images.githubusercontent.com/36241930/104355019-10962a80-5530-11eb-8ac3-b94711baee10.png">